opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -88,9 +90,21 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
     let logger_provider = init_logs().unwrap();
 
     // Create a new OpenTelemetryLogBridge using the above LoggerProvider.
-    let otel_log_appender = OpenTelemetryLogBridge::new(&logger_provider);
-    log::set_boxed_logger(Box::new(otel_log_appender)).unwrap();
-    log::set_max_level(Level::Info.to_level_filter());
+    let layer = OpenTelemetryTracingBridge::new(&logger_provider);
+
+    // add a tracing filter to filter the events generated from the crates used by opentelemetry-otlp
```

Comment:
please add a link to the tracking issue, and also warn that, the events from this layer will be filtered for this layer irrespective of if they occur inside OTLP Exporting or not, potentially leading to blind spots.